ui/src/tasks/actions/index.ts

```diff
@@ -458,7 +458,7 @@ export const importTask = (script: string) => async (
 export const getRuns = (taskID: string) => async (dispatch): Promise<void> => {
   try {
     dispatch(setRuns([], RemoteDataState.Loading))
-
+    await dispatch(selectTaskByID(taskID))
```

Comment:
Could these be done in parallel?
```
const [runs] = await Promise.all([
  client.tasks.getRunsByTaskID(taskID)
  dispatch(selectTaskByID(taskID))
])
```

Reply:
Yes, they don't depend on each other, I updated it